components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/APIProviderImpl.java

```diff
@@ -698,9 +698,10 @@ public void updateEntitlementPolicies(List<EntitlementPolicy> policies) throws
      *
      * @param policyId Entitlement policy id
      * @return entitlement policy content
+     * @throws AppManagementException
      */
     @Override
-    public String getEntitlementPolicy(String policyId) {
+    public String getEntitlementPolicy(String policyId) throws AppManagementException {
```

Comment:
Formatting issues and Java doc comment issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files